specification/hybridconnectivity/resource-manager/readme.md

```diff
@@ -101,7 +101,6 @@ This is not used by Autorest itself.
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
   - repo: azure-sdk-for-net-track2
```

Comment:
Is the idea to eventually remove all other languages so tooling generates the SDK from TypeSpec?